openrest4j-api/src/main/java/com/openrest/v1_1/CurbsideInfo.java

```diff
@@ -0,0 +1,53 @@
+package com.openrest.v1_1;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+
+import java.io.Serializable;
+import java.util.*;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CurbsideInfo implements Serializable, Cloneable {
+    private static final long serialVersionUID = 1L;
+
+    /** Default constructor for JSON deserialization. */
+    public CurbsideInfo() {
+        this.additionalInformationRequired = false;
+    }
+
+    public CurbsideInfo(String instructions, Boolean withAdditionalInformation, String additionalInformation) {
+        this.instructions = instructions;
+        this.additionalInformationRequired = (withAdditionalInformation != null) ? withAdditionalInformation : false;
```

Comment:
simple assign is enough - no need for != null

Reply:
done